feat: add Qualcomm IoT page

huwshimi opened the pull request:
## Done

- Add Qualcomm IoT page and contact form.

Copy doc for the webpage: https://docs.google.com/document/d/1BP2T63R6kdnUFQFIZcbXbTIgY-9fQVgVRLsSUU5Rvx8/edit?tab=t.0
Copy doc for the form: https://www.google.com/url?q=https://docs.google.com/document/d/1vZepBdXUWbidF3BD1zwvudiykRjtItHT4ssCqj6Dh-U/edit?tab%3Dt.0&sa=D&source=docs&ust=1731909684091428&usg=AOvVaw3Gjv_xqSPKCnQoYfDiXgwx

## QA

- Open the demo https://ubuntu-com-14500.demos.haus/
- Check that there is a link to the new page:
 -- In the top nav click 'Get Ubuntu'.
 -- To the side click 'Ubuntu for IoT'
 -- Under 'Quick links' there should be a link: 'Install Ubuntu on Qualcomm IoT Platforms'.
- Visit https://ubuntu-com-14500.demos.haus/download/qualcomm-iot
- Click 'Get in touch' and the new form should appear.

## Issue / Card

https://warthogs.atlassian.net/browse/WD-16036

## Screenshots

![iot](https://github.com/user-attachments/assets/73e484c9-1464-403b-8282-12740758d0df)

![Screenshot 2024-11-26 at 14-48-26 Install Ubuntu on Qualcomm IoT Platforms Ubuntu](https://github.com/user-attachments/assets/ced7910a-b696-4ac9-b54e-78618c413f9e)

webteam-app commented:
### [<img src='https://assets.ubuntu.com/v1/6baef514-ubuntu-circle-of-friends-large.svg' height=32 width=32> Demo</img>](https://ubuntu-com-14500.demos.haus)
### [<img src='https://assets.ubuntu.com/v1/e512b0e2-jenkins.svg' height=32 width=32> Jenkins </img>](https://jenkins.canonical.com/webteam/job/start-demo/4457)
### [<img src='https://assets.ubuntu.com/v1/7144ec6d-logo-jaas-icon.svg' height=32 width=32> demos.haus </img>](https://demos.haus)

akbarkz commented on templates/download/qualcomm-iot.html:
```diff
@@ -0,0 +1,166 @@
+        <p>
+          If you have any questions about the platform or would like information
+          about our certification program, <a
+          href="/download/contact-us#contact-us" class="p-button--positive
```
this should be a simple link, according to design, not a button

huwshimi replied:
Oops! Thanks, fixed.

huwshimi commented on templates/download/qualcomm-iot.html:
```diff
@@ -0,0 +1,166 @@
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <a href="https://www.qualcomm.com/">
+          {{ image (
```
Fixed.

huwshimi commented on templates/download/qualcomm-iot.html:
```diff
@@ -0,0 +1,166 @@
+            <h3 class="p-heading--5">QCS6490</h3>
+          </div>
+          <div class="col-6 u-vertically-center u-align--center p-image-wrapper is-product">
+            {{ image (
```
Done.

huwshimi commented on templates/download/qualcomm-iot.html:
```diff
@@ -0,0 +1,164 @@
+            </p>
+            <hr class="p-rule--muted" />
+            <p>
+              Ubuntu Server 22.04 for QCS6490 <a href="TODO">Release Notes (TODO)</a>
```
The release notes link needs to be added here.

mattea-turic commented:
Thanks @huwshimi ! Just a couple comments:

- Could you add shallow padding below "Ubuntu Server 22.04 for QSCS6490 Release Notes" that's in the section "QCS6490 Qualcomm RB3 Gen 2 Vision kit" under "get started"?
<img width="500" alt="Screenshot 2024-11-26 at 09 18 12" src="https://github.com/user-attachments/assets/603ac051-9920-46d9-a66d-9d911141bdb3">

- There also looks to be too much bottom padding for that section too (looks like 3*24px) – could this just be regular pls/
- Could you add a `</br>` after "a" in the section "Are you working on a commercial project?"
- There should also be deep bottom padding
<img width="500" alt="Screenshot 2024-11-26 at 09 22 06" src="https://github.com/user-attachments/assets/79d8ca07-4f70-48cf-883f-ad97ac221044">

eliman11 commented:
Thanks @huwshimi for your work on this! A few comments:
- [ ] [Reach out to Canonical](https://ubuntu.com/internet-of-things/qualcomm/contact-us?_gl=1*nj12mt*_gcl_au*ODIyNTkyNDY0LjE3MjY4NTExMzc.&_ga=2.36841312.152565662.1732614045-1500419473.1726851136) in the last section is returning a 404, I have a question out to the stakeholders on the copydoc for a revised link. Will let you know when there's any update
- [ ]  Alt text currently missing for the image, also posted a question out to stakeholders.
- [ ]  Also defering to @mattea-turic but not sure if the text on the button should wrap  like this on mobile?
<img width="367" alt="Screenshot 2024-11-26 at 13 51 48" src="https://github.com/user-attachments/assets/7a555761-b865-492c-afb9-71976f8b6824">

eliman11 commented:
> Thanks @huwshimi for your work on this! A few comments:
> 
> * [ ]  [Reach out to Canonical](https://ubuntu.com/internet-of-things/qualcomm/contact-us?_gl=1*nj12mt*_gcl_au*ODIyNTkyNDY0LjE3MjY4NTExMzc.&_ga=2.36841312.152565662.1732614045-1500419473.1726851136) in the last section is returning a 404, I have a question out to the stakeholders on the copydoc for a revised link. Will let you know when there's any update
> * [ ]   Alt text currently missing for the image, also posted a question out to stakeholders.
> * [ ]   Also defering to @mattea-turic but not sure if the text on the button should wrap  like this on mobile?
> 
> <img alt="Screenshot 2024-11-26 at 13 51 48" width="367" src="https://private-user-images.githubusercontent.com/181769624/389982991-7a555761-b865-492c-afb9-71976f8b6824.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI2MzA1MDYsIm5iZiI6MTczMjYzMDIwNiwicGF0aCI6Ii8xODE3Njk2MjQvMzg5OTgyOTkxLTdhNTU1NzYxLWI4NjUtNDkyYy1hZmI5LTcxOTc2ZjhiNjgyNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyNlQxNDEwMDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMDcwNzU2MTkxYjg0NmEyNGIwMjhlOTgzYWQ0NWU3MjI3YTA4Zjg5NjgxMTI2Mjg1MzUzNGM2ZmU4NWNlMjk0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ihL0x8L_rezE4mjS0Grtyy8qF1z0lgIvk4ENT0lTCQc">

eliman11 commented:
Given the urgency of this task please feel free to add UX+1 if you're just waiting for me and I don't hear back re the link and alt text. We can fix these later so I'm not a blocker!

mattea-turic commented:
> Thanks @huwshimi for your work on this! A few comments:
> 
> * [ ]  [Reach out to Canonical](https://ubuntu.com/internet-of-things/qualcomm/contact-us?_gl=1*nj12mt*_gcl_au*ODIyNTkyNDY0LjE3MjY4NTExMzc.&_ga=2.36841312.152565662.1732614045-1500419473.1726851136) in the last section is returning a 404, I have a question out to the stakeholders on the copydoc for a revised link. Will let you know when there's any update
> * [ ]   Alt text currently missing for the image, also posted a question out to stakeholders.
> * [ ]   Also defering to @mattea-turic but not sure if the text on the button should wrap  like this on mobile?
> 
> <img alt="Screenshot 2024-11-26 at 13 51 48" width="367" src="https://private-user-images.githubusercontent.com/181769624/389982991-7a555761-b865-492c-afb9-71976f8b6824.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI2MzMyMDEsIm5iZiI6MTczMjYzMjkwMSwicGF0aCI6Ii8xODE3Njk2MjQvMzg5OTgyOTkxLTdhNTU1NzYxLWI4NjUtNDkyYy1hZmI5LTcxOTc2ZjhiNjgyNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyNlQxNDU1MDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zNjg5NjlhZmUwOTFjNGUyOTA5MzcyNGI4ZTI0NDc3OTZiMTBmZTEzY2E3MjJkMTY2ZDI0YjRhNWUxMTM0NTBjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Jv_ckiDZs5cT5b4laXI93qrfT3P2SdaPJrSgDXFibuo">

Button is correct, this is how it wraps in vanilla ([example](https://vanillaframework.io/docs/examples/base/button?theme=light))
<img width="349" alt="Screenshot 2024-11-26 at 15 49 11" src="https://github.com/user-attachments/assets/bb09fc92-d5e9-4a63-9e84-574c54278406">

eliman11 commented:
Thanks for the clarification @mattea-turic! You can ignore my last bullet point in that case. 

- [ ]  One additional quick fix - would you be able to change the field type for the phone number to the one we have with a preset drop-down for country codes? [Example here](https://canonical.com/#get-in-touch)
<img width="1408" alt="Screenshot 2024-11-26 at 13 36 03" src="https://github.com/user-attachments/assets/64b7e50b-175d-424e-af83-59c80f7207a2">

huwshimi commented:
> Thanks @huwshimi ! Just a couple comments:
> 
> * Could you add shallow padding below "Ubuntu Server 22.04 for QSCS6490 Release Notes" that's in the section "QCS6490 Qualcomm RB3 Gen 2 Vision kit" under "get started"?
> * There also looks to be too much bottom padding for that section too (looks like 3*24px) – could this just be regular pls/

These two should be fixed:

<img width="1346" alt="Screenshot 2024-11-27 at 2 57 44 pm" src="https://github.com/user-attachments/assets/a57f22f2-37f9-4fb4-85c9-2749370dd711">



> * Could you add a `</br>` after "a" in the section "Are you working on a commercial project?"

Done. I've adjusted this as the screen gets smaller otherwise the title breaks like this:

<img width="645" alt="Screenshot 2024-11-27 at 3 00 41 pm" src="https://github.com/user-attachments/assets/e8e0451f-d6f0-4a6f-afca-dc9a896b1242">

<img width="556" alt="Screenshot 2024-11-27 at 3 01 29 pm" src="https://github.com/user-attachments/assets/804d1eb3-a913-4baf-a62b-7831ad97f897">

So I've adjusted the smaller screens to work just break the title naturally:

<img width="646" alt="Screenshot 2024-11-27 at 3 01 01 pm" src="https://github.com/user-attachments/assets/6902948c-0ecf-4b9b-bea0-608e487417ab">

<img width="558" alt="Screenshot 2024-11-27 at 3 01 16 pm" src="https://github.com/user-attachments/assets/4bc81182-49c9-45e7-9daa-689d343767a4">

Is that OK?

> * There should also be deep bottom padding

Done:

<img width="1161" alt="Screenshot 2024-11-27 at 3 17 06 pm" src="https://github.com/user-attachments/assets/b137b7f8-ea15-449a-bde3-0894d5cf3ff8">

huwshimi commented:
> Thanks for the clarification @mattea-turic! You can ignore my last bullet point in that case.
> 
> * [ ]   One additional quick fix - would you be able to change the field type for the phone number to the one we have with a preset drop-down for country codes? [Example here](https://canonical.com/#get-in-touch)
> 

It looks like this feature isn't currently working for modal forms, I've filed an issue here: https://github.com/canonical/ubuntu.com/issues/14509.

eliman11 commented:
Noted on the forms, thank you for escalating! Received a reply from the stakeholder re the following
> * [ ]  [Reach out to Canonical](https://ubuntu.com/internet-of-things/qualcomm/contact-us?_gl=1*nj12mt*_gcl_au*ODIyNTkyNDY0LjE3MjY4NTExMzc.&_ga=2.36841312.152565662.1732614045-1500419473.1726851136) in the last section is returning a 404, I have a question out to the stakeholders on the copydoc for a revised link. Will let you know when there's any update

Could you please change the link here to the same [form in the hero CTA](https://ubuntu-com-14500.demos.haus/download/qualcomm-iot#get-in-touch) 

> * [ ]   Alt text currently missing for the image, also posted a question out to stakeholders.

Please add the alt text 'Qualcomm® RB3 Gen 2 Development Kit'. Also marked this in the copydoc.

huwshimi commented:
> Noted on the forms, thank you for escalating! Received a reply from the stakeholder re the following
> 
> > * [ ]   [Reach out to Canonical](https://ubuntu.com/internet-of-things/qualcomm/contact-us?_gl=1*nj12mt*_gcl_au*ODIyNTkyNDY0LjE3MjY4NTExMzc.&_ga=2.36841312.152565662.1732614045-1500419473.1726851136) in the last section is returning a 404, I have a question out to the stakeholders on the copydoc for a revised link. Will let you know when there's any update
> 
> Could you please change the link here to the same [form in the hero CTA](https://ubuntu-com-14500.demos.haus/download/qualcomm-iot#get-in-touch)

Done.

> 
> > * [ ]    Alt text currently missing for the image, also posted a question out to stakeholders.
> 
> Please add the alt text 'Qualcomm® RB3 Gen 2 Development Kit'. Also marked this in the copydoc.

Done.
